unit_tests/stereotype_client_tests.js

```diff
@@ -19,6 +19,26 @@ let templBody = 'Hello {{name}}.';
 let templateType = 'text/x-handlebars-template';
 let contentType = 'text/x-handlebars-template; charset=utf-8; postProcessor=mjml';
 
+function getSampleTemplateResource(templateId, contentType) {
+  let info = {
+    templateId: templateId,
+    links: {
+      self: {
+        href: `${StereotypeOptions.baseUrl}/v1/templates/${templateName}`,
+      },
+    },
+  };
+
+  if (contentType) {
+    info.contentType = contentType;
+  }
+
+  return info;
+}
+
+function equalJson(expected, received) {
```

Comment:
minor nitpick: assert.deepEqual has a similar signature (`assert.deepEqual(received, expected)`) and belongs to a built-in module. There is also assert.deepStrictEqual.

Reply:
Removed equalJson().